pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+              ``mode="axis"`` instead.
+
+        x_pointing : list, optional
+            List of the ``[x, y, z]`` coordinates specifying the X axis
+            pointing in the global coordinate system if ``mode="axis"``.
+            The default is ``[1, 0, 0]``.
+        y_pointing : list, optional
+            List of the ``[x, y, z]`` coordinates specifying the Y axis
+            pointing in the global coordinate system if ``mode="axis"``.
+            The default is ``[0, 1, 0]``.
+        phi : float, optional
+            Euler angle phi in degrees if ``mode="zxz"`` or ``mode="zyz"``.
+            The default is ``0``.
+        theta : float, optional
+            Euler angle theta or rotation angle in degrees if ``mode="zxz"``,
+            ``mode="zyz"``, or ``mode="axisrotation"``. The default is ``0``.
+        psi : float, optional
+            Euler angle psi in degrees if ``mode="zxz"`` or ``mode="zyz"``.
+            The default is ``0``.
+        u : list
+            List of the ``[ux, uy, uz]`` coordinates for the rotation axis
+            if ``mode="zxz"``. The default is ``[1, 0, 0]``.
+
+        Returns
+        -------
+        :class:`pyaedt.modeler.Modeler.CoordinateSystem`
+            Created coordinate system.
+
+        References
+        ----------
+
+        >>> oEditor.CreateRelativeCS
+        """
+        if name:
+            cs_names = [i.name for i in self.coordinate_systems]
+            if name in cs_names:
+                raise AttributeError("A coordinate system with the specified name already exists!")
+
+        cs = CoordinateSystem(self)
+        if cs:
+            result = cs.create(
+                origin=origin,
+                reference_cs=reference_cs,
+                name=name,
+                mode=mode,
+                view=view,
+                x_pointing=x_pointing,
+                y_pointing=y_pointing,
+                phi=phi,
+                theta=theta,
+                psi=psi,
+                u=u,
+            )
+            if result:
+                return cs
+        return False
+
+    @pyaedt_function_handler()
+    def create_face_coordinate_system(
+        self, face, origin, axis_position, axis="X", name=None, offset=None, rotation=0, always_move_to_end=True
+    ):
+        """Create a face coordinate system.
+        The face coordinate has always the Z axis parallel to face normal.
+        The X and Y axis lie on the face plane.
+
+        Parameters
+        ----------
+        face : int, FacePrimitive
+            Face where the coordinate system is defined.
+        origin : int, FacePrimitive, EdgePrimitive, VertexPrimitive
+            Specify the coordinate system origin. The origin must belong to the face where the
+            coordinate system is defined.
+            If a face is specified, the origin is placed on the face center. It must be the same as ``face``.
```

Comment:
```suggestion
            
            - If a face is specified, the origin is placed on the face center. It must be the same as ``face``.
```
Making all the "If" sentences list items so that are rendered better in the doc.